ssr/index.js

```diff
@@ -218,9 +218,24 @@ function walk(directory, callback) {
 
 function renderDocuments(
   paths,
-  { buildHtml, output, quiet, ...args },
+  { buildHtml, output, quiet, noProgressBar, ...args },
   returnDocumentBuilt = false
 ) {
+  /**
+   * It's more useful to display either progress bar or each built file, but not both.
+   * But if `--quiet` is passed, we should disable both
+   *
+   * @see https://github.com/mdn/stumptown-renderer/pull/259#pullrequestreview-325419683
+   */
+  const useProgressBar =
+    !quiet &&
+    (JSON.parse(process.env.CI || "false") ||
```

Comment:
Should that be...

```
const useProgressBar =
    !quiet &&
    !(
      JSON.parse(process.env.CI || "false") ||
      !process.stdout.isTTY ||
      !noProgressBar
    );
```
...instead?
I.e. a negation on the whole bracket. 
It definitely makes my local testing with `CI=true yarn deployment-build` work as expected, which is that it lists every single file.

Reply:
Crap. I was wrong. Ignore me for now. I think you can figure it out. 

Actually, way to test it is:
```
yarn workspace ssr build && CI=true node cli/index.js ssr --build-html
# or 
yarn workspace ssr build && CI=false node cli/index.js ssr --build-html
# or
yarn workspace ssr build && node cli/index.js ssr --build-html --quiet
# or
yarn workspace ssr build && node cli/index.js ssr --build-html --no-progress-bar
```
```

Reply:
If I change it to:
```javascript
const useProgressBar =
    !quiet &&
    !noProgressBar &&
    (!JSON.parse(process.env.CI || "false") || !process.stdout.isTTY);
```
the above 4 scenarios works as expected.

Reply:
Sorry for not testing this enough. Thanks for the flawless logic! :grin: